freqtrade/rpc/telegram.py

```diff
@@ -330,8 +330,12 @@ def _status_table(self, update: Update, context: CallbackContext) -> None:
             statlist, head = self._rpc._rpc_status_table(
                 self._config['stake_currency'], self._config.get('fiat_display_currency', ''))
 
-            message = tabulate(statlist, headers=head, tablefmt='simple')
-            self._send_msg(f"<pre>{message}</pre>", parse_mode=ParseMode.HTML)
+            max_trades_per_msg = 50
+            for i in range(0, max(int(len(statlist) / max_trades_per_msg), 1)):
```

Comment:
in principle great - except that it doesn't work / send the 2nd message.

I think the problem is in this line - you're limiting the upper range to 1 (`max(x, 1)`) - so at maximum, 1 message can be sent.

Also, in case of 52 trades (for example) - i'd have `52 / 50 = 1.04` - using "int" on this is 1 ... while in reality, i'd need 2 (`range(0, 1)` will iterate only once).

I think what you'd need instead is something like `range(0, max(int(len((statlist) / max_trades_per_msg) + 1), 1))` ...

Reply:
You're absolutely right, well spotted! I changed the code, and verified on my machine that the status table is properly sent when using:
* 20 trades -> 1 messages
* 54 trades -> 2 messages
* 104 trades -> 3 messages